### ISSUE TITLE
Bug, or misunderstanding? Screen Label not in Exported metadata

### ISSUE THREAD

**Issue by PhotoJoseph** [hvdwolf/jExifToolGUI#272]
Running macOS 12.4, Intel, Java 1.0 (11.0.10) (and I just ran Java 8 Update 351 to be sure). jExifToolGUI is 2.0.1

I'm trying to create a metadata export that I can import into DaVinci Resolve. I'm really close to making this work as I want it! Resolve requires specific headers in the CSV file on import, of course. I was expecting that when I made a custom user-defined metadata set, that the "screen label" would be exported in the CSV as the header with the "Metadata tags" column being the content, but that's not the case. 

If that's not the way it's supposed to work, then please consider that a feature request! 

I'd also like to eliminate certain fields that are exported. There's a row at the top that says "x image files read", and the first column is "SourceFile" with the file path. If I could eliminate those, and have the custom name in the header, then I wouldn't need to alter the file at all before importing to Resolve!

![CleanShot 2022-10-24 at 18 20 42@2x](https://user-images.githubusercontent.com/78106443/197659870-7b81b633-9979-4ff6-9b4a-a8c1069a515c.png)

**Comment by hvdwolf** [comment]
The "x images read" is certainly a bug. I need to fix that.
The other is a feature request. 
And what do you mean with elminate those? As you talk about the "x images read" and "SourceFile".Do you want the column removed or do you want to be able to change the name of SourceFile into "something"?
How will your program name what to do without reference to the image?

**Comment by PhotoJoseph** [comment]
I would like to remove that row and the SourceFile column, yes. Perhaps an option to "only include exported metadata" would be the solution; i.e. the app doesn't add anything? 

As to how the files get matched, in my case there's a metadata field I extract that has the file name, and doesn't include the path, which is what I need. So I'd extract that FileName field, move it to the front (which is the issue I'm having with the other  issue, #271 — I can't move it where I want it!), and that makes a complete CSV for me.

**Comment by hvdwolf** [comment]
Yes, indeed.  Custom headers above your columns, and also the option to select which colums will be exported is a feature request. It is also an extremely difficult feature request. The csv export is an exiftool functionality. I can't influence that functionality. It would require a lot of programming on my side.

**Comment by PhotoJoseph** [comment]
oh I see. The custom column export I'm handling with "user defined metadata set" (minus the extra columns/header), so I can totally get the job done, it just requires an extra step. If that's not going to happen, no worries, just good to know not to wait around for it ;-)

**Comment by mrtngrsbch** [comment]
@PhotoJoseph.
Note that jExifToolGUI is a 'GUI' of ExifTool. 
These features you are requesting I would suggest you look for them in the ExifTool community or create some Python script yourself, like this one: https://github.com/MuseosAbiertos/Museum-Metadata-Embedder

**Comment by PhotoJoseph** [comment]
Understood, @mrtngrsbch — as a non-engineer I don't necessarily know what features are part of the GUI and which are part of the main app. I've managed to work around the issues in a way that works for me, so I've made great progress.
 
<img width="511" alt="CleanShot 2022-10-25 at 14 05 44@2x" src="https://user-images.githubusercontent.com/78106443/197887603-b81b9973-120a-4fd4-8173-5d1fd540dfe9.png">

**Comment by mrtngrsbch** [comment]
@PhotoJoseph,
Yo también tengo necesidades particulares que escapan de jExifToolGUI y que nunca estarán disponibles. Si bien puede no resultar fácil, con ExifTool y la linea de comandos puedes lograr resultados exóticos y precisos.
Mira esto: https://adamtheautomator.com/exiftool/
o esto otro: https://ninedegreesbelow.com/photography/exiftool-commands.html

**Comment by PhotoJoseph** [comment]
Thanks @mrtngrsbch I will check those out.

**Comment by hvdwolf** [comment]
I am a long time exiftool user (12+ years) and recent versions (but that is already more than 3 years :) ) support the specifications of which tags you wnt to be exported. However, there is no way whatsoever to get rid of the first "SourceFile" header and column data.
I will put this on my "ToDo" list, but it is not high priority for me.

**Comment by hvdwolf** [comment]
Forgot to mention that I did fix the "x images read". For the time being I will not build the custom headers.
